frontend/components/App/FishbowlPreJoin/form-auth.tsx

```diff
@@ -7,6 +7,7 @@
  * file that was distributed with this source code.
  */
 
+import React from 'react';
```

Comment:
With nextjs we don't need to import it but, in case we need it, we should do it this way according this[ microsoft's guide](https://github.com/Microsoft/TypeScript-React-Conversion-Guide#typescript-react-conversion-guide). 

```suggestion
import * as React from 'react';
```